### PR TITLE
Fix the index "Featured Chapter" blockquote #296

### LINKED ISSUE #296: Featured Chapter blockquote
Fix the index "Featured Chapter" blockquote as right now it uses an image called `quote.png` with a dimension of 8192x7973 px :/

Proposed solution: replace quote.png with a text CSS pseudo element and get rid of that HTTP request completely.

### PR DESCRIPTION
**Pull request by catalinred**
This fixes #296, #300 and #301

### CODE REVIEW

**Comment by rviscomi** [comment]
Thanks for working on this! I'll need to spend some time testing this locally to make sure the style changes don't break anything.

**Comment by catalinred** [comment]
Sure, please let me know if you have any questions!

File: src/templates/en/2019/base.html [hunk 1]
```diff
@@ -79,28 +80,28 @@
...
 
-function toggleMenu(e) {
-  document.body.classList.toggle('menu-open');
-}
-
-menuIcon.addEventListener('click', toggleMenu);
-menuCloseIcon.addEventListener('click', toggleMenu);
-})();
+        document.querySelector('.menu-btn').addEventListener('click', function() {
+          document.body.classList.toggle('menu-open');
```
Comment (rviscomi) [inline src/templates/en/2019/base.html]:
```suggestion
          var menuOpen = document.body.classList.toggle('menu-open');
          menuIcon.src = menuOpen ? menuIconSrcClose : menuIconInitialSrc;
```

**Comment by rviscomi** [comment]
Also feel free to add yourself to the `developers` team in your 2019.json config!

**Pull request merged**